index.js

```diff
@@ -0,0 +1,247 @@
+  <a href="https://github.com/">see source <img src="/assets/VectorDD.png" alt=""></a>
+  </div>
+  </div>
+  </div>
+</div>
+</div>`;
+const btn0d = document.getElementById('seeprojecty');
+const btn1d = document.getElementById('seeprojectyy');
+const spand = document.getElementsByClassName('closeD')[0];
+function popupd() {
+  modalD.style.display = 'block';
+}
+function closseed() {
+  modalD.style.display = 'none';
+}
+spand.onclick = function () {
+  closseed();
+};
+
+// form validation
+const formField = document.querySelector('.upper-form-V');
+const userName = document.querySelector('.name-V');
+const emailinput = document.querySelector('.email-V');
+const textArea = document.querySelector('.message-V');
+const errorMessage = document.querySelector('.error-message');
+const reg = /^([a-z0-9_\-.]+)@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,3})$/;
+formField.addEventListener('submit', (events) => {
+  events.preventDefault();
+  if (userName.value.trim() === '') {
+    errorMessage.style.display = 'block';
+    errorMessage.textContent = 'name can not be empty';
+  } else if (emailinput.value.trim() === '') {
+    errorMessage.style.display = 'block';
+    errorMessage.textContent = 'email can not be empty';
+  } else if (!emailinput.value.match(reg)) {
+    emailinput.style.borderColor = 'red';
+    errorMessage.style.display = 'block';
+    errorMessage.textContent = 'email is not valid, please enter a valid email all in lower case';
+  } else if (textArea.value.trim() === '') {
+    errorMessage.style.display = 'block';
+    errorMessage.textContent = 'text area can not be empty';
+  } else {
+    errorMessage.style.display = 'none';
+    formField.submit();
+  }
+});
+
+// form validation desk
+const formFiel = document.querySelector('.upper-form-desk');
+const userNamed = document.querySelector('#named');
+const emailinputd = document.querySelector('#emaild');
+const textAread = document.querySelector('#messages');
+const errorMessaged = document.querySelector('.error-messaged');
+const regd = /^([a-z0-9_\-.]+)@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,3})$/;
+formFiel.addEventListener('submit', (events) => {
+  events.preventDefault();
+  if (userNamed.value.trim() === '') {
+    errorMessaged.style.display = 'block';
+    errorMessaged.textContent = 'name can not be empty';
+  } else if (emailinputd.value.trim() === '') {
+    errorMessaged.style.display = 'block';
+    errorMessaged.textContent = 'email can not be empty';
+  } else if (!emailinputd.value.match(regd)) {
+    emailinputd.style.borderColor = 'red';
+    errorMessaged.style.display = 'block';
+    errorMessaged.textContent = 'Email is not valid, please enter a valid email all in lower case';
+  } else if (textAread.value.trim() === '') {
+    errorMessaged.style.display = 'block';
+    errorMessaged.textContent = 'text area can not be empty';
+  } else {
+    errorMessaged.style.display = 'none';
+    formFiel.submit();
```

Comment:
When the user email input is uppercase, it is noticing the user well and not submitting the form 👍  But when you used lowercases it is not submitting the form as well. Kindly fix that.
<img width="1149" alt="Screenshot 2023-03-18 at 3 53 35 PM" src="https://user-images.githubusercontent.com/25887253/226107693-cc8b9c06-269c-4cbe-a795-fc34c7285ec3.png">